google-cloud-contrib/google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProviderTest.java

```diff
@@ -655,28 +654,26 @@ public void testNewFileSystem() throws IOException {
   }
 
   @Test
-  public void testFromSpace() throws UnsupportedEncodingException {
+  public void testFromSpace() throws UnsupportedEncodingException, IOException {
```

Comment:
Went through every single test in this file and changed the exception type to Exception. Removed unneeded imports.